### ISSUE TITLE
NaN or Inf found in input tensor.

### ISSUE THREAD

**Issue by vfishc** [JOP-Lee/READ#32]
Thanks for your excellent work!!!

But during running the codes in the folder src with the data offered by you(because I don't have a monitor), the evaluate loss are NAN. The printed logs are:

![image](https://user-images.githubusercontent.com/84083106/225273450-86633b47-7772-45eb-a345-8f5185809bbd.png). If any opearations of mine are wrong or there are some issues in the data or code?

Best wishes

**Comment by JOP-Lee** [comment]
I think your training data is too small to be divided into test sets

**Comment by vfishc** [comment]
But I use the data and config files offered by the authors? What should I revise to include all of the training data?

Thanks~

**Comment by booker-max** [comment]
> But I use the data and config files offered by the authors? What should I revise to include all of the training data?
> 
> Thanks~

Hi, I encountered the same problem as you, I also used the data and config files provided by the author, did you solve the problem?

**Comment by yixionghuo** [comment]
Hi, I had the same problem as you, did you solve it?

**Comment by baekhyun77** [comment]
Hi，Has anyone solved this problem?

**Comment by zZH1222hui** [comment]
Did you solve this problem?

**Comment by dyzxzeng** [comment]
The data is not placed in the right path. You can refer to the path.yaml.

**Comment by VerseWei** [comment]
Hello, I am sure I placed the sample data in the path indicated by path.yaml, but I still encountered this issue. How did you do it? Can you explain in detail? Thank you @dyzxzeng

**Comment by VerseWei** [comment]
I am in the same situation as you. Have you solved this problem? @booker-max
